addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -226,6 +217,7 @@ registerModel({
                 currentRtcSession: clear(),
                 disconnectAudioMonitor: clear(),
                 logs: clear(),
+                rtcDataChannels: clear(),
```

Comment:
I don't think we want this field on rtc, this should be automatically done when deleting the related sessions.

Reply:
there is no `clear()` of all `RtcSession`s here. Only the `currentRtcSession`.
So I'm not sure whether removing this LOC is safe.